### PR TITLE
Topic pills redesign

### PR DESCRIPTION
**Pull request by oetherington**
This PR adds the new topic pill styles:
Screenshot 2023-03-07 at 11 12 04
Screenshot 2023-03-07 at 11 12 31

And hovered:
Screenshot 2023-03-07 at 11 12 11
Screenshot 2023-03-07 at 11 12 19
Screenshot 2023-03-07 at 11 12 24

I also updated the close button as I saw this was on the todo list and it was trivial to add here:
Screenshot 2023-03-07 at 11 16 00

Two open questions:

- Do we need to adjust the sizing of the icons. In the first screenshot I posted the "Wild Animal Welfare" icon looks about the right size to me, but "Biosecurity" may be a little to big. Doing this is a little fiddly, so I figured I'd get another opinion before wasting a load of time on it.
- Do we want to make any changes to the core topics section on the new posts page? We could just add the new core topic styles, but we'd need some way to distinguish between a selected core tag and an unselected core tag:
Screenshot 2023-03-07 at 11 18 19

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204123390082356) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
nice one, this looks really great 👏 👏  

> Do we need to adjust the sizing of the icons. In the first screenshot I posted the "Wild Animal Welfare" icon looks about the right size to me, but "Biosecurity" may be a little to big. Doing this is a little fiddly, so I figured I'd get another opinion before wasting a load of time on it.

yup I think we need to do that. May be better for the svgs to be the same size though instead of you tweaking the code for each one? I could export and update them if you think that'd be better

> Do we want to make any changes to the core topics section on the new posts page? We could just add the new core topic styles, but we'd need some way to distinguish between a selected core tag and an unselected core tag:

good question! If it's not too much work I would go for new core topic styles with a 60% opacity for those not selected

I would also either decrease the height of the pills _or_ increase the width (usually there is a bit more padding on the sides than top/bottom–in the designs I have 4px top/bottom and 8px left/right)

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 1]
```diff
@@ -192,8 +192,6 @@ const PostsPagePostHeader = ({post, answers = [], toggleEmbeddedPlayer, hideMenu
     <AnalyticsContext pageSectionContext="topSequenceNavigation">
       <PostsTopSequencesNav post={post} />
     </AnalyticsContext>
-    {!post.group && !post.sequence && !post.question && <PostsPageTopTag post={post} />}
```
Comment (Will-Howard) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:195]: Are we just getting rid of this? Was this in the designs or did you just infer it?

**Review by Will-Howard — APPROVED** [review]
Some of the icons look a bit close to the left border to me, maybe add a bit of padding?
Screenshot 2023-03-08 at 11 12 29

Otherwise this looks great, approving subject to design review

**Comment by oetherington** [comment]
@agnesstenlund Great! If it's quick and easy to reexport the svgs in the same size then that's probably the best long-term solution, but let me know if it's too much work.

I'll go ahead and add the extra padding and unselected styles.

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx (hunk 1 above)
Comment (oetherington) [reply packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:195]: Yep - it's in the designs 🙂

**Comment by oetherington** [comment]
@agnesstenlund I've just added the updated icons (thanks again for re-exporting!):
Screenshot 2023-03-09 at 18 31 18
Screenshot 2023-03-09 at 18 31 14

I also added a little bit of padding on the icon, but I wouldn't be surprised if that needs a little bit of tweaking.

For testing, I also created the post [http://localhost:3000/posts/oe7kFXbbvmFFsnqgM/test-locking-comments](http://localhost:3000/posts/oe7kFXbbvmFFsnqgM/test-locking-comments) in the dev database which has all of the core tags applied to it.

Here are the new styles for selected/unselected core tags in the new post page options:
Screenshot 2023-03-09 at 18 35 38
Screenshot 2023-03-09 at 18 35 32

**Comment by agnesstenlund** [comment]
really nice! seeing this there are still a couple ones I'd like to tweak that I missed–will send those over 👍

**Comment by agnesstenlund** [comment]
Also, I think we just have "Animal welfare" as a core topic, not farmed and wild? 🤔 At least according to the topics page 
Screenshot 2023-03-10 at 11 56 47

**Comment by agnesstenlund** [comment]
here they are! I doubt they have the right names so I added this image to show which one should go where:
Screenshot 2023-03-10 at 13 31 00

SVGs below 👇 
aiSafetyIcon
causePrioritisationIcon
communityGlobeIcon
groupsIcon
policyIcon

**Comment by oetherington** [comment]
@agnesstenlund Great - thanks! I've just added the new icons:
Screenshot 2023-03-13 at 11 49 55
Screenshot 2023-03-13 at 11 49 44

Let me know if you think they need any tweaks, otherwise I think this branch is done.

Also, it looks like you're correct about it just being "Animal welfare" rather than separate tags for "wild" and "farmed" - it's just old data in the dev database. On prod, animal welfare uses the same icon as wild animal welfare in the screenshots above.

**Comment by agnesstenlund** [comment]
looks great! 🚢

**Pull request merged**